Do not use /usr/include for headers

Using /usr/include is wrong when doing cross-compilation, so get rid
of it, and rely on the compiler to use the appropriate default paths
for headers.

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>

Index: b/setup.py
===================================================================
--- a/setup.py
+++ b/setup.py
@@ -370,7 +370,7 @@
       'ext_modules': plat_ext + [
             # _fastmath (uses GNU mp library)
             Extension("Crypto.PublicKey._fastmath",
-                      include_dirs=['src/','/usr/include/'],
+                      include_dirs=['src/'],
                       libraries=['gmp'],
                       sources=["src/_fastmath.c"]),
 
